### PR TITLE
Fix Pathfinder distribution issue

### PR DESCRIPTION
**Pull request by jstortoise**
- Updated chainId for getPools in TokenSwapDistribution

### CODE REVIEW

File: src/client/js/components/partials/swap/TokenSwapDistribution.jsx [hunk 1]
```diff
@@ -21,7 +21,8 @@ export default class TokenSwapDistribution extends Component {
   }
 
   componentDidMount() {
-    PathFinder.getPools(1).then((pools) => this.setState({ pools }));
+    const { chainId = 1 } = TokenListManager.getCurrentNetworkConfig() || {};
+    PathFinder.getPools(chainId).then((pools) => this.setState({ pools }));
```
Comment (chillvybz) [inline src/client/js/components/partials/swap/TokenSwapDistribution.jsx:25]: will any chainId gracefully work here?
We want it to fail gracefully for non-Pathfinder chains

Comment (jstortoise) [reply src/client/js/components/partials/swap/TokenSwapDistribution.jsx]: @chillvybz Fixed. This will only work for Pathfinder's supported chain ids

**Comment by chillvybz** [comment]
Routing Distribution still broken for ETH, Optimism and Arbritum:

See screenshots. Also seems like the incoming Exchange names basically don't have any proper identifiers to pull up the Exchange icons either. 

Screen Shot 2022-03-18 at 2 50 40 PM
Screen Shot 2022-03-18 at 2 52 09 PM
Screen Shot 2022-03-18 at 2 52 22 PM

**Comment by jstortoise** [comment]
> Routing Distribution still broken for ETH, Optimism and Arbritum:
> 
> See screenshots. Also seems like the incoming Exchange names basically don't have any proper identifiers to pull up the Exchange icons either.
> 
@chillvybz paraswap and 1inch doesn't provider proper identifiers for Exchange names. We can get icons from 1inch exchanges list but paraswap doesn't provide any identifiers or icons. Please take a look at paraswap screenshot. It provides only Exchange names

Screen Shot 2022-03-21 at 10 36 27 AM

**Pull request merged**